roles/deploy/tasks/nfs.yml

```diff
@@ -0,0 +1,24 @@
+---
+
+- name: install nfs-utils
+  yum: name=nfs-utils state=latest
+  vars:
+    nfsmounts:
+      - {path: "/home/scud/downloadchd/email", src: "ipo-file-svm-lif-be1.internal.ch:chd_image"}
+      - {path: "/home/scud/downloadceu/email", src: "ipo-file-svm-lif-be1.internal.ch:chcc_email"}
+      - {path: "/home/scud/downloadwck/email", src: "ipo-file-svm-lif-be1.internal.ch:wck_image"}
+      - {path: "/home/scud/downloadceu/post", src: "ipo-file-svm-lif-be1.internal.ch:chcc_post"}
+      - {path: "/data1/apps/scud", src: "ipo-file-svm-lif-be1.internal.ch:scanning"}
+      - {path: "/data1/home/scanning", src: "ipo-file-svm-lif-be1.internal.ch:tiffexport"}
+      - {path: "/data1/image/scud", src: "ipo-file-svm-lif-be1.internal.ch:tiffexport"}
+
```

Comment:
The `nfsmounts` variable here will be ignored as it isn't used by the `yum` module. We'll need to define this elsewhere, either as a role default or as a host/group variable. The latter would be preferable in this case, as mount points will likely need to be scoped to each environment. We can achieve this by using tags to target specific environments in our dynamic inventory. For example, to define mount points for the `live` environment:

In `group_vars/tag_Environment_live.yml`

```yaml
nfsmounts:
  - path: "/home/scud/downloadchd/email"
    src: "ipo-file-svm-lif-be1.internal.ch:chd_image"
  - path: "/home/scud/downloadceu/email"
    src: "ipo-file-svm-lif-be1.internal.ch:chcc_email"
  - path: "/home/scud/downloadwck/email"
    src: "ipo-file-svm-lif-be1.internal.ch:wck_image"
  - path: "/home/scud/downloadceu/post"
    src: "ipo-file-svm-lif-be1.internal.ch:chcc_post"
  - path: "/data1/apps/scud"
    src: "ipo-file-svm-lif-be1.internal.ch:scanning"
  - path: "/data1/home/scanning"
    src: "ipo-file-svm-lif-be1.internal.ch:tiffexport"
  - path: "/data1/image/scud"
    src: "ipo-file-svm-lif-be1.internal.ch:tiffexport"
```

The dynamic inventory configuration files will also need to be updated to introduce a new keyed group. For example, in `inventory/heritage_live_aws_ec2.yml`:

```yaml
keyed_groups:
  - key: tags.Environment
    prefix: tag_Environment_
    separator: ""
```